src/error.rs

```diff
@@ -50,6 +50,9 @@ pub enum Error {
     /// Failed to create a new endpoint.
     #[error("Creating endpoint")]
     Endpoint(#[from] quinn::EndpointError),
+    /// Failed to set/get priority of stream.
+    #[error("Unknown stream, cannot set/get priority.")]
+    Priority,
```

Comment:
Will renaming this to `UnknownStream` make more sense? Since that's the only possible error in this scenario